src/mscorlib/src/System/String.Manipulation.cs

```diff
@@ -946,7 +946,95 @@ public String Remove(int startIndex, int count)
             Contract.EndContractBlock();
 
             return Substring(0, startIndex);
-        }   
+        }
+
+        public string Replace(string oldValue, string newValue, bool ignoreCase, CultureInfo culture)
```

Comment:
that is fine. can we just add assert here? I am asking for that because in the future if anyone added any code here can assume non null value if not looking at ReplaceCore.

---
In reply to: [101003861](https://github.com/dotnet/coreclr/pull/9316#discussion_r101003861) [](ancestors = 101003861)